README.md

```diff
@@ -81,16 +83,12 @@ func main() {
 
 	// Change part/parts of user
 	changes := map[string]string{"name": "Jane"}
-	if _, err = db.Change(selectedUser.ID, changes); err != nil {
```

Comment:
Why was this removed?

Reply:
This was not working and giving a panic hence removed it. I can add it if we have figured out why it was not working.

Reply:
in #98 it has been changed to modify.
Try the main branch instead of the latest release.